services/docker-compose.devel.yml

```diff
@@ -66,17 +66,42 @@ services:
       - ../packages:/devel/packages
       - ${ETC_HOSTNAME:-/etc/hostname}:/home/scu/hostname:ro
     environment:
-      # force to start as cpu mode otherwise it will boot in gpu mode in development
-      - START_AS_MODE_CPU=1
-      - SC_BOOT_MODE=debug-ptvsd
-      - SIDECAR_LOGLEVEL=DEBUG
+      &dev-sidecar-environment # force to start as cpu mode otherwise it will boot in gpu mode in development
+      START_AS_MODE_CPU: 1
+      SC_BOOT_MODE: debug-ptvsd
+      SIDECAR_LOGLEVEL: DEBUG
     ports:
       - "3002:3000"
-    deploy:
-      # NOTE: Allows 3002 to be exposed for ptvsd
+    deploy: &dev-sidecar-deploy # NOTE: Allows 3002 to be exposed for ptvsd
```

Comment:
this NOTE is for the endpoint_mode right?

Reply:
yes. it was already there